### DISCUSSION TITLE
Cpu usage on the fade effect

### DISCUSSION THREAD

**Discussion by Bonswally** [4JX/L5P-Keyboard-RGB#87] [General]
Cool tool, but is it normal to be getting 3-6% background cpu usage on a 5800h when using the fade preset?  On the other presets it seems to be lower.  Makes the fans run at idle.

Also it's annoying how I have to manually select my profile every time I start the software, I'm sure there's a way around it that I've missed.

**Comment by 4JX** [comment]
It "is" with how the code is set up right now as it checks  every few ms, but it can probably be lowered with a small sleep between checks (at the cost of responsiveness)

**Reply by Bonswally** [reply]
Oh ok, so it's just how it has to be with that type of effect.  What a shame.  

Thanks.

**Comment by 4JX** [comment]
> Also it's annoying how I have to manually select my profile every time I start the software, I'm sure there's a way around it that I've missed.

Save your profile as a file named default.json, never got around to documenting it exists
